### PR TITLE
BDCAT RELEASE 1594132572

### PR DESCRIPTION
**Pull request by themarcelor**
copying files from preprod.gen3.biodatacatalyst.nhlbi.nih.gov to gen3.biodatacatalyst.nhlbi.nih.gov

**DO NOT FORGET TO:**

- [x] Load in indexd database from Preprod
- [x] Load in sheepdog database from Preprod
- [x] Load in metadata-service database from Preprod
- [ ] Run ETL
- [ ] QA Smoke Test (Data access for S3 and Google, PFB handoff, Workspaces)
- [ ] Merge Data-Manifest PR: https://github.com/nhlbidatastage/Data-Manifest/pull/22

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

File: gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json [hunk 1]
```diff
@@ -152,15 +152,15 @@
...
         "type": "subject"
       },
       {
-        "index": "dcp_file",
+        "index": "gen3.biodatacatalyst.nhlbi.nih.gov_file",
         "type": "file"
       }
     ],
-    "config_index": "dcp_array-config",
+    "config_index": "gen3.biodatacatalyst.nhlbi.nih.gov_array-config",
```
Comment (emalinowski) [inline gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json:163]: This should be gen3.biodatacatalyst.nhlbi.nih.gov_subject-array-config.

**Comment by PlanXCyborg** [comment]
# gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

**Comment by PlanXCyborg** [comment]
# gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)

**Pull request merged**